### ISSUE TITLE
Template tab_menu_navbar_header oddly make blank title

### ISSUE THREAD

**Issue by Yippy** [sonata-project/SonataAdminBundle#4421]
#### Sonata packages

```
sonata-project/admin-bundle              3.14.0 The missing Symfony Admin Generator                        
sonata-project/block-bundle              3.3.2  Symfony SonataBlockBundle                                  
sonata-project/cache                     1.0.7  Cache library                                              
sonata-project/core-bundle               3.3.0  Symfony SonataCoreBundle                                   
sonata-project/datagrid-bundle           2.2.1  Symfony SonataDatagridBundle                               
sonata-project/doctrine-orm-admin-bundle 3.1.4  Symfony Sonata / Integrate Doctrine ORM into the Sonat...  
sonata-project/exporter                  1.7.1  Lightweight Exporter library                               
sonata-project/intl-bundle               2.3.0  Symfony SonataIntlBundle 
```

#### Symfony packages

```
symfony/monolog-bundle     v3.0.3 Symfony MonologBundle                                                    
symfony/phpunit-bridge     v3.2.6 Symfony PHPUnit Bridge                                                   
symfony/polyfill-apcu      v1.3.0 Symfony polyfill backporting apcu_* functions to lower PHP versions      
symfony/polyfill-intl-icu  v1.3.0 Symfony polyfill for intl's ICU-related data and classes                 
                                                                                                           
symfony/polyfill-php56    symfony/polyfill-mbstring  v1.3.0 Symfony polyfill for the Mbstring extension v1.
3.0 Symfony polyfill backporting some PHP 5.6+ features to lower PHP ver...                                
symfony/polyfill-php70     v1.3.0 Symfony polyfill backporting some PHP 7.0+ features to lower PHP ver...  
symfony/polyfill-util      v1.3.0 Symfony utilities for portability of PHP codes                           
symfony/security-acl       v3.0.0 Symfony Security Component - ACL (Access Control List)                   
symfony/swiftmailer-bundle v2.5.4 Symfony SwiftmailerBundle                                                
symfony/symfony            v3.2.6 The Symfony PHP framework   
```

#### PHP version

```bash
PHP 7.0.13-0ubuntu0.16.04.1 (cli) ( NTS )                                                                  
Copyright (c) 1997-2016 The PHP Group                                                                      
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies                                              
    with Zend OPcache v7.0.13-0ubuntu0.16.04.1, Copyright (c) 1999-2016, by Zend Technologies 
```

## Subject

There are empty spacing in  _navbar_title which display blank header in mobile view

```
{% block tab_menu_navbar_header %}
    {% if _navbar_title is not empty %}
        <div class="navbar-header">
            <a class="navbar-brand" href="#">{{ _navbar_title|raw }}</a>
        </div>
     {% endif %}
 {% endblock %}`
```

fix by trim check
```
{% block tab_menu_navbar_header %}
    {% if _navbar_title|trim %}
        <div class="navbar-header">
            <a class="navbar-brand" href="#">{{ _navbar_title|raw }}</a>
        </div>
    {% endif %}
 {% endblock %}
```

probably best that the _navbar_title doesn't return spaces or enters.

## Results
Left is real result, right is expected result.
![bug2](https://cloud.githubusercontent.com/assets/6518915/24316003/a90464ee-10e2-11e7-9c49-4b7899e64baf.png)

**Comment by core23** [comment]
Can you please provide a PR for that? @Yippy

**Comment by Yippy** [comment]
looks like @jordisala1991 fixed it with his PR #4408, line 22 has trim
